experimental/perf/scripts/bench_run_log.py

```diff
@@ -139,16 +144,27 @@ def run_experiment(
         help="length of metric names.",
         default=10,
     )
-
     parser.add_argument(
         "-o",
         "--outfile",
         type=str,
         help="performance test result output file.",
         default="results.json",
     )
+    parser.add_argument(
+        "-d",
+        "--data_type",
+        type=str,
+        help="wandb data type to log. Default scalar.",
+        default="scalar",
+    )
```

Comment:
maybe make it a choice so users don't need to guess the name:
https://docs.python.org/3/library/argparse.html#choices

Reply:
yep